documentation/staging/content/samples/azure-kubernetes-service/includes/create-aks-cluster-storage.txt

```diff
@@ -1,86 +1,147 @@
-#### Create storage and set up file share
+#### Create storage
 
 Our usage pattern for the operator involves creating Kubernetes "persistent volumes" to allow the WebLogic Server to persist its configuration and data separately from the Kubernetes Pods that run WebLogic Server workloads.
 
 We will create an external data volume to access and persist data. There are several options for data sharing as described in [Storage options for applications in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/azure/aks/concepts-storage).
 
-We will use Azure Files as a Kubernetes volume. For details about this full featured cloud storage solution, see the [Azure Files Documentation](https://docs.microsoft.com/azure/aks/azure-files-volume).
+We will dynamically create and use a persistent volume with Azure Files NFS share.  For details about this full featured cloud storage solution, see the [Azure Files Documentation](https://docs.microsoft.com/en-us/azure/aks/azure-files-dynamic-pv).
 
-##### Create an Azure Storage account
+##### Create an Azure Storage account and NFS share
 
-Create a storage account using the Azure CLI. Note that the storage account name can contain only lowercase letters and numbers, and must be between 3 and 24 characters in length:
+1. Create an Azure Storage Account
 
-```shell
-# Change the value as needed for your own environment
-$ export AKS_PERS_STORAGE_ACCOUNT_NAME="${NAME_PREFIX}storage${TIMESTAMP}"
-
-$ az storage account create \
-   -n $AKS_PERS_STORAGE_ACCOUNT_NAME \
-   -g $AKS_PERS_RESOURCE_GROUP \
-   -l $AKS_PERS_LOCATION \
-   --sku Standard_LRS
-```
+    Create a storage account using the Azure CLI. Make sure the following values must be specified:
```

Comment:
Please make the same edits here that I suggested previously for these statements.

Reply:
See https://github.com/galiacheng/weblogic-kubernetes-operator/commit/2b18f06d51752b28a22ec323464107e1d3305788